jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/SingleProjectIntegrationTest.java

```diff
@@ -170,6 +178,39 @@ public void testBuild_simple() throws IOException, InterruptedException, DigestE
     assertWorkingDirectory("/home", targetImage);
   }
 
+  @Test
+  public void testDockerDaemon_simpleOnJava11()
+      throws DigestException, IOException, InterruptedException {
+    Assume.assumeTrue(isJava11RuntimeOrHigher());
+
+    String targetImage = "localhost:6000/simpleimage:gradle" + System.nanoTime();
+    Assert.assertEquals(
+        "Hello, world. \n",
+        JibRunHelper.buildToDockerDaemonAndRun(
+            simpleTestProject, targetImage, "build-java11.gradle"));
+  }
+
+  @Test
+  public void testDockerDaemon_simpleWithIncompatibleJava11()
+      throws DigestException, IOException, InterruptedException {
+    Assume.assumeTrue(isJava11RuntimeOrHigher());
+
+    try {
+      JibRunHelper.buildToDockerDaemonAndRun(
+          simpleTestProject, "willnotbuild", "build-java11-incompatible.gradle");
+      Assert.fail();
+
+    } catch (UnexpectedBuildFailure ex) {
+      Assert.assertThat(
+          ex.getMessage(),
+          CoreMatchers.containsString(
+              "The base image uses Java 8, but project is using Java 11, perhaps you should "
```

Comment:
Trivial tweak:
> *your* project

But I wonder if flipping the order helps identify the mismatch:
> _Your project is using Java 11 but the base image is for Java 8.  Perhaps…_

Reply:
Sounds better. Unfortunately, `, perhaps you should` is the part automatically added by `HelpfulSuggestions.suggest()`, so can't fix here to use a dot.